internal/net/control/control.go

```diff
@@ -41,6 +42,7 @@ type control struct {
 	keepAlive                int
 }
 
+// SockerFlag represent the flag to enable specific feature for the socket listener.
```

Comment:
I think that "represents" is more correct representation grammatically.

Reply:
```suggestion
// SocketFlag represents the flag to enable specific feature for the socket listener.
```